crates/turborepo-lib/src/run/global_hash.rs

```diff
@@ -64,12 +67,23 @@ pub fn get_global_hash_inputs<'a, L: ?Sized + Lockfile>(
     let mut global_deps = HashSet::new();
 
     if !global_file_dependencies.is_empty() {
-        let globs = package_manager.get_workspace_globs(root_path)?;
+        let exclusions = match package_manager.get_workspace_globs(root_path) {
```

Comment:
Technically this is a little less efficient than the Go version because we're getting the inclusions even though we don't need them. But that's a minor cost since regardless we have to deserialize the workspaces, process the globs, etc.